consensus/raft/consensus.go

```diff
@@ -229,6 +229,24 @@ func (cc *Consensus) Ready(ctx context.Context) <-chan struct{} {
 	return cc.readyCh
 }
 
+// IsTrustedPeer returns true. In Raft we trust all peers.
+//
+// TODO: There is a potential improvement here by just trusting peers which
+// are currently part of the Raft peerset. But we cannot call Peers() every
+// single time and should keep a cache of trusted peers indexed by peer which
+// is difficult to maintain when peers are removed (we get no
+// notification). All in all, since PeerAdd is effectively open to everyone in
+// Cluster, it makes little difference.
```

Comment:
I think for now only use-case for raft is where the cluster is private and all nodes are handled by one entity. Unless, some new use case come up, I think we should just trust all raft peers.

Reply:
ok